### ISSUE TITLE
explicit instruction to start autotest

### ISSUE THREAD

**Issue by tansaku** [saasbook/hw-oracle-of-bacon#10]
Autotest didn't work for me on 2.3.0 or 2.2.2 on c9 - although I think it has worked for others  (on their local systems)

As per earlier assignments I think we should suggest running individual tests via the line number

**Comment by lokeshh** [comment]
How about using [guard-rspec](https://github.com/guard/guard-rspec)?

Guard seems to be more popular than autotest.

**Comment by tansaku** [comment]
@lokeshh yeah might work - although to be honest I'm not a fan of these automated things and I think they are particularly confusing for newcomers.

There's always the question about whether the thing has run with the latest code or not ...

**Comment by tansaku** [comment]
in the past (this May) I found it easier to get autotest running than guard ...

**Comment by lokeshh** [comment]
Can't we just include the `Guardfile` so one doesn't need to do any configuration.

**Comment by tansaku** [comment]
oh sure - not saying it's that tricky to configure - just that I couldn't get it to work on c9 in May, but autotest just ran.  Autotest now just doesn't run, and pedagogically I think learners are better off running rspec spec/spec_file.rb:17 and pinpointing what they are doing.

@armandofox 's a fan of autotest and is upgrading to Guard - if you want to do a PR that would be welcome

**Comment by lokeshh** [comment]
It have sent a PR. It was very easy to setup the Guard. All I needed to do was `bundle exec guard init rspec` and it worked.
